server/helpers/mfaBannerType.js

```diff
@@ -0,0 +1,9 @@
+const EXISTING_USER = 'EXISTING_USER'
+const NEW_USER = 'NEW_USER'
+const FIRST_TIME_USER = 'FIRST_TIME_USER'
+
+module.exports = Object.freeze({
```

Comment:
was more expecting this to be typescript instead of javascript - see https://www.typescriptlang.org/docs/handbook/enums.html#string-enums